addons/website/static/src/client_actions/website_preview/website_preview.js

```diff
@@ -345,7 +345,9 @@ export class WebsitePreview extends Component {
                 // as to avoid DOM changes not handled by the editor.
                 // (Such as clicking on a link that triggers navigating to
                 // another page.)
-                ev.preventDefault();
+                if (!(ev.target.matches && ev.target.matches('.o_edit_menu_popover a'))) {
```

Comment:
> I went with the first one but IMO it has the drawback of having this check on every click, it might slow things down.

It's the same with the other suggestions you made, even if it's internally in jQuery/the browser :wink: 

I would go with this too but:
- Why do you needed to check `matches`? `target` is always an element node no?
- Instead: `if (!ev.target.closest('#oe_manipulators') {` :wink: More generic and that's actually the point of the manipulators stuff :+1:

Reply:
> Why do you needed to check matches? target is always an element node no?

I think you're right. in the case of a click event, I don't think it can be anything but an element. It was just because the type was not inferred  in my IDE (since Event.target is an EventTarget which can be anything) which made me question if it would be there all the time. I don't think a check is needed.